src/Directory.Build.props

```
@@ -7,8 +7,8 @@
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <PackageOutputPath>..\bin\$(Configuration)</PackageOutputPath>
     <PackageProjectUrl>https://bonsai-rx.org/machinelearning</PackageProjectUrl>
-    <IncludeSymbols Condition="'$(Configuration)'=='Release'">true</IncludeSymbols>
-    <GeneratePackageOnBuild Condition="'$(Configuration)'=='Release'">true</GeneratePackageOnBuild>
+    <IncludeSymbols Condition="'$(Configuration)'=='Debug'">true</IncludeSymbols>
+    <GeneratePackageOnBuild Condition="'$(Configuration)'=='Debug'">true</GeneratePackageOnBuild>
```

```suggestion
    <IncludeSymbols>true</IncludeSymbols>
    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
```

If it is useful to have symbols and packages on both Release and Debug configurations, the easiest approach would be to simply eliminate the condition.